package.json

```diff
@@ -129,7 +137,7 @@
         "FACEBOOK_HYBRID_APP_EVENTS": "false",
         "FACEBOOK_ADVERTISER_ID_COLLECTION": "true",
         "FACEBOOK_ANDROID_SDK_VERSION": "17.0.0",
```

Comment:
```suggestion
        "FACEBOOK_ANDROID_SDK_VERSION": "18.0.2",
```

Reply:
Sorry but this is the actual working config! Please create a PR with your suggestion to test it separately 🧪 
The Facebook SDK was failing during the login request and theses configs make it work.